code/modules/food_and_drink/snacks.dm

```diff
@@ -807,12 +807,12 @@ ABSTRACT_TYPE(/obj/item/reagent_containers/food/snacks/soup)
 	prize = 0
 
 /obj/item/reagent_containers/food/snacks/cereal_box/cirrial
-	name = "cereal box -'Flocked-Flakes Cirrial'"
-	desc = "A bluey-green cereal that beeps gently at you, they're grrrrowing out of the box oh fuck!"
```

Comment:
this is a frosted flakes/frosties reference, can probably stay in.

Reply:
Lol I didn't even realize that the name and description was one big Frosted Flakes reference. The description is back to how it was.